### PR TITLE
Bluetooth: controller: llcp: pause data during PHY update procedure

### PR DESCRIPTION
**Pull request by kruithofa**
This PR fixes the PHY update procedures for conformance tests when
being a Central by pausing data transmission during the PHY update 
procedure
    
To be more precise, following tests are fixed:
    LL/CON/CEN/BV-49-C
    LL/CON/CEN/BV-50-C
    LL/CON/CEN/BV-53-C
    LL/CON/CEN/BV-54-C

Unittests are also updated
    

The 3rd commit avoids regression errors on the Peripheral PHY tests:

LL/CON/PER/BV-49-C
LL/CON/PER/BV-50-C
LL/CON/PER/BV-52-C
LL/CON/PER/BV-53-C
LL/CON/PER/BV-54-C
LL/CON/PER/BV-55-C
LL/CON/PER/BV-56-C
LL/CON/PER/BV-58-C

Signed-off-by: Andries Kruithof <andries.kruithof@nordicsemi.no>

### CODE REVIEW

**Review by cvinayak — DISMISSED** [review]

File: subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c [hunk 1]
```diff
@@ -887,7 +889,8 @@ static void rp_pu_send_phy_update_ind(struct ll_conn *conn, struct proc_ctx *ctx
 				      void *param)
 {
 	if (llcp_rr_ispaused(conn) || !llcp_tx_alloc_peek(conn, ctx) ||
-	(llcp_rr_get_paused_cmd(conn) == PROC_PHY_UPDATE)) {
+	(llcp_rr_get_paused_cmd(conn) == PROC_PHY_UPDATE) ||
+	    !ull_is_lll_tx_queue_empty(conn)) {
```
Comment (erbr-ot) [inline subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c:894]: why this? Means phy update will not add tx unless tx queue is empty? That seems wrong?

Comment (erbr-ot) [reply subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c:894]: Gottit

**Review by erbr-ot — DISMISSED** [review]
LGTM

**Review by thoh-ot — DISMISSED** [review]

**Review by ppryga-nordic — CHANGES_REQUESTED** [review]
One nit-pick and one other request: Can you update commit messages to give some information what was wrong and what is fixed, except IDs of conformance tests?

**Review by ppryga-nordic — APPROVED** [review]

**Review by sjanc — APPROVED** [review]

**Pull request merged**